### PR TITLE
Ps/subclass task argument

### PR DESCRIPTION
**Pull request by Pastromhaug**

### CODE REVIEW

File: mbq/ranch/killswitch.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    class KillSwitchTask(celery.Task):
+        def killswitch_name(self):
+            service = settings.RANCH["service"]
+            return f"task-killswitch-{service}-{self.name}".replace(".", "-").replace(
+                "_", "-"
+            )
+
+        def __call__(self, *args, **kwargs):
+            if variation(self.killswitch_name(), False):
+                from . import _collector
```
Comment (mattbriancon) [inline mbq/ranch/killswitch.py:22]: Why do we import here and at the top of the module?

Comment (Pastromhaug) [reply mbq/ranch/killswitch.py:22]: because this import can't happen until django is initialized. The collector comes from app.ready

Comment (Pastromhaug) [reply mbq/ranch/killswitch.py:22]: And the collector comes from app.ready because it uses tags from settings.RANCH

Comment (Pastromhaug) [reply mbq/ranch/killswitch.py:22]: fixed

File: mbq/ranch/__init__.py [hunk 2]
```diff
@@ -8,5 +8,6 @@
     __version__,
 )
 
+from .killswitch import create_killswitch_task_class  # noqa
```
Comment (mattbriancon) [inline mbq/ranch/__init__.py]: OK I'm just gonna say it. We don't need to talk about it.

But how nice would it be if this was `from . import killswitch` and then you'd use it with

```python
from mbq import ranch
ranch.killswitch.create_task_class(foo.variation)
```

Comment (Pastromhaug) [reply mbq/ranch/__init__.py]: I like this, made the change.

**Review by mattbriancon — APPROVED** [review]

**Pull request merged**